Update wdog.h

yuhaim opened the pull request:
wdog.func may be called by passing only one parameter. As is in nuttx/sched/wdog/wd_start.c
         switch (wdog->argc)
            {
              default:
                DEBUGPANIC();
                break;

              case 0:
                (*((wdentry0_t)(wdog->func)))(0);
                break;

davids5 commented:
@chenming5828 General submissions to Nuttx are done via patches submitted to @nuttx 

Please see https://groups.yahoo.com/neo/groups/nuttx/info

yuhaim committed:
Update wdog.h

wdog.func may be called by passing only one parameter. As is in nuttx/sched/wdog/wd_start.c
         switch (wdog->argc)
            {
              default:
                DEBUGPANIC();
                break;

              case 0:
                (*((wdentry0_t)(wdog->func)))(0);
                break;

yuhaim commented:
Sorry for this duplicated pull request. 
But Gregory Nutt declined bruenn's request in the upstream project, maybe this is a more convincing reason to fix it?
[https://bitbucket.org/nuttx/nuttx/issues/159/sched-wdog-fix-signature-for-wdentry_t](url)

dagar commented:
The master branch on this repo tracks Nuttx upstream.

davids5 commented:
In Nuttx 10.0.0
